Emergence/Contexts/App Setup/Auth and Slideshow/SlideshowView.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+/// Takes a [String] as imagePaths and goes through them all
+
+class SlideshowView: UIView {
+    var imagePaths: [String] = []
+    private var index = -1
+
+    private var imageView: UIImageView {
```

Comment:
This is a computed property, so wouldn't accessing it will invoke this code every time, creating a new `UIImageView` and adding it to your view hierarchy? I think a `lazy` property assigned to a closure would fit better.